src/js/components/form-elements/TableBodyVirtualized.jsx

```diff
@@ -28,18 +31,20 @@ class TableBodyVirtualized extends Component {
 
   getHeight() {
     const { fieldsConfig: { subfieldKey, getDynamicRowAttr }, fields, properties } = this.props;
-    const { totalCount } = properties;
     let height = 0;
     const maxTableHeight = window.innerHeight < 900 ?
       // 0.35 * window.innerHeight = 35vh from table-content class in StockMovement.scss
       0.35 * window.innerHeight : 0.40 * window.innerHeight;
 
     if (!subfieldKey) {
-      if (totalCount * 28 > maxTableHeight) {
-        height = maxTableHeight;
-      } else if (totalCount > 0) {
-        height = totalCount * 28;
-      }
+      height = fields.value.reduce((acc, field) => {
```

Comment:
@kchelstowski have you tested this against cases where there were more items on the page (around 20+)? Plus, have you tested if there is no regression on substituting items, split picking or split packing?